client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -82,6 +83,16 @@
      * @param checkpoint The checkpoint to restore to.
      */
     void resetReadersToCheckpoint(Checkpoint checkpoint);
+
+    /**
+     * Restore the reader group to the provided {@link StreamCut StreamCut(s)}. All readers in the group will encounter
+     * a {@link ReinitializationRequiredException} and when they rejoin the group they will resume from the position
+     * provided by the stream cuts.
+     *
+     * @param streamCuts {@link StreamCut} for every stream managed by the reader group.
+     */
+    @Beta
+    void resetReadersToStreamCut(Set<StreamCut> streamCuts);
```

Comment:
This could probably be unified with the above API.

Reply:
done.